Code_Exercises/Exercise_06_Vector_Add/solution.cpp

```diff
@@ -30,13 +30,7 @@ TEST_CASE("vector_add", "vector_add_solution") {
   }
 
   try {
-    auto asyncHandler = [&](sycl::exception_list exceptionList) {
-      for (auto& e : exceptionList) {
-        std::rethrow_exception(e);
-      }
-    };
-
-    auto defaultQueue = sycl::queue{sycl::default_selector{}, asyncHandler};
+    auto defaultQueue = sycl::queue{};
```

Comment:
You've also gotten rid of the sycl::default_selector{} here, which I guess ought to remain?